torch/csrc/jit/script/python_tree_views.cpp

```diff
@@ -38,11 +38,7 @@ struct SourceRangeFactory {
       int line,
       int start_col,
       int end_col) {
-    // Python has a weird convention where col_offset points to the column
```

Comment:
Is this not true? Or was this here just to fix the other bug where `pos` in `source_range.h` pointed to the `\n` character?

Reply:
It appears that it is not true. I think all previous versions of this were starting the line with the `\n` so it was thought that things were off-by-1.